src/helpers/format.ts

```diff
@@ -105,3 +108,30 @@ export const formatBytes = (numBytes: number): string => {
     ? `${(numBytes / 1024 ** 2).toFixed(1)} MB`
     : `${(numBytes / 1024 ** 3).toFixed(1)} GB`;
 };
+
+export function getLogLevelCharts(
```

Comment:
Please write a test for this function. It's rather long. Maybe you can break it down after writing tests for it.